### ISSUE TITLE
wrong on dynamic nested folder

### ISSUE THREAD

**Issue by hi-reeve** [victorgarciaesgi/nuxt-typed-router#85]
i have nested dynamic folder like this `/:id/:document`
but this leads to wrong typed route
the generated routes is like this 
```ts
 'admin-master-data-vessel-id-document-create': {
        id: (string | number)[];
  };
```

where its supposed to be like this
```ts
 'admin-master-data-vessel-id-document-create': {
        id: string | number;
       document: string | number;
  };
```

**Comment by hi-reeve** [comment]
additional info, this happen if we use the latest nuxt 3.4.1

**Comment by hi-reeve** [comment]
here's repro https://stackblitz.com/edit/nuxt-starter-ffmx85?file=.nuxt/typed-router/__routes.ts

**Comment by victorgarciaesgi** [comment]
Yeah seems like they made a breaking change on how the paths are generated

**Comment by victorgarciaesgi** [comment]
Was tricky but support for nuxt `3.4.x` is available in beta `3.1.4-beta.0`
I will test on older nuxt version to see if it's still compatible

**Comment by hi-reeve** [comment]
any eta this will be move to 3.4.1?
